### DISCUSSION TITLE
XCH tokens did not come to the CHIA light wallet

### DISCUSSION THREAD

**Discussion by RAkot13** [Chia-Network/chia-blockchain#14238] [Support]
Hello, I ran into a problem: when withdrawing XCH from the MEXC exchange, 2,181 XCH tokens got stuck, 6 days have passed since the withdrawal, the exchange transferred xch, There is a Transaction Hash with withdrawal confirmation, How to solve this problem? 
Here is the TXID:
https://xchscan.com/zh/txns/0x78e344cb8311ef496aaa050d367145f1ca6a9bbd48596cbda726c9770570d6a2

**Comment by Bodman1** [comment]
> Hello, I ran into a problem: when withdrawing XCH from the MEXC exchange, 2,181 XCH tokens got stuck, 6 days have passed since the withdrawal, the exchange transferred xch, There is a Transaction Hash with withdrawal confirmation, How to solve this problem?
> Here is the TXID:
> https://xchscan.com/zh/txns/0x78e344cb8311ef496aaa050d367145f1ca6a9bbd48596cbda726c9770570d6a2

I have the same problem, how did you solve yours?
